app/facades/npc_facade.rb

```diff
@@ -1,8 +1,100 @@
 require 'faker'
 class NPCFacade
-  def self.create_npc(score_type)
+  def self.create_npc(score_type, level)
     npc_ancestry = NPCService.create_npc_ancestry
     npc_class = NPCService.create_npc_class
-    NPC.new(npc_ancestry, npc_class, score_type)
+    base_info = NPC.new(npc_ancestry, npc_class, score_type, level)
+    npc = [NpcModel.create!(alignment: base_info.alignment,
```

Comment:
I'm sure there's a better way of doing this. I'm doing some research to see how.